README.md

```diff
@@ -11,8 +11,30 @@ organisation of neuroimaging data.
 In this repository, we develop the
 [BIDS specification](https://bids-specification.readthedocs.io/en/latest/).
 
-**Want to learn more about working with BIDS? Have a question, comment, or suggestion?**
-Open or comment on one of our [NeuroStars issues](https://neurostars.org/tags/bids) or check out the [BIDS Starter Kit](https://github.com/bids-standard/bids-starter-kit)!
+# When to use BIDS
+
+To organize your data in BIDS, all you need is neuro data, a computer, and the
+[BIDS specification](https://bids-specification.readthedocs.io/en/stable/).
+
+BIDS currently supports MRI, MEG, EEG, iEEG, behavioral, and physiological data
+and more modalities are being added continuously (see here for
+[work in progress modalities](https://bids-specification.readthedocs.io/en/stable/06-extensions.html#bids-extension-proposals)).
+
+# Formatting your data with BIDS
+
+As a dataset curator, the GitHub repository and all files therein can be safely ignored.
+Users should focus on [the rendered content](https://bids-specification.readthedocs.io/en/stable/).
+The specification is provided in the form of a webpage, built using
+[MkDocs](https://www.mkdocs.org/) and [Read the Docs](https://readthedocs.org/).
+
+*Want to learn more about working with BIDS? Have a question, comment, or suggestion?*
```

Comment:
Light suggestion here would be to convert this into a sub-heading, or add one directly above it. Something like:

```
## Where to go if you need help
```